GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -822,12 +790,12 @@ - (void)testDisconnect_refreshToken {
   [self verifyAndRevokeToken:kRefreshToken hasCallback:YES];
   [_authorization verify];
   [_authState verify];
-  [_tokenResponse verify];
+  XCTAssertNil([_keychainHandler loadAuthState]);
 }
 
 // Verifies disconnect errors are passed along to the callback.
 - (void)testDisconnect_errors {
-  [[[_authorization expect] andReturn:_authState] authState];
+  [_keychainHandler saveAuthState:_authState];
```

Comment:
Wouldn't we expect the `_keychainHandler`'s `authState` to be nil upon disconnecting? If so, I recommend that we test that here and throughout this test case.

Reply:
You could then probably remove lines like this: `[_authState verify];`.

Reply:
> Wouldn't we expect the `_keychainHandler`'s `authState` to be nil upon disconnecting? If so, I recommend that we test that here and throughout this test case.

There should be an `authState` in the keychain since there is an error. I added XCTAssertNotNil to make it explicit.



> You could then probably remove lines like this: `[_authState verify];`.

[_authState verify] is to verify the line
`[[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];`

The pattern in OCMock is `expect and verify`. Do you think we should use OCMStub instead here?

Reply:
That's a good suggestion. My plan is to do that in a separate PR. This PR is all about `GIDKeychainHandler`.